action.yml

```diff
@@ -47,7 +47,15 @@ runs:
           if [[ ${{ inputs.cosign-release }} == 'v0.6.0' ]]; then
             # v0.6.0's linux release has a dependency on `libpcsclite1`
             sudo apt-get update -q
-            sudo apt-get install -yq libpcsclite1
+            set +e
+            sudo dpkg -s libpcsclite1
+            if [ $? -eq 0 ]; then
+                echo "INFO: libpcsclite1 package is already installed"
+            else
+                 echo "INFO: libpcsclite1 package is not installed, installing it now."
+                 sudo apt-get install -yq libpcsclite1
```

Comment:
should we do the spt update here as well?

Reply:
of course, it makes sense, thank you @cpanato 🙏